railties/test/engine_test.rb

```diff
@@ -24,4 +24,14 @@ def initialize
     end
     assert_equal "world", klass.instance.hello
   end
+
+  test "application object isn't output when a NoMethorError is raised" do
```

Comment:
Shouldn't we be testing this at the railtie level?

Reply:
It is checking with an application though, I think we need to either use an engine or a railtie and move this file in this case.